tests/test_math/test_backend_manager.py

```diff
@@ -460,24 +468,27 @@ def test_is_trainable(self):
         arr1 = np.array([1, 2])
         arr2 = tf.constant(arr1)
         arr3 = tf.Variable(arr1)
+        arr4 = jnp.array(arr1)
 
         assert not math.is_trainable(arr1)
         assert not math.is_trainable(arr2)
         assert math.is_trainable(arr3) is (math.backend_name == "tensorflow")
+        if math.backend_name == "jax":
+            assert not (math.is_trainable(arr4))
```

Comment:
```suggestion
            assert not math.is_trainable(arr4)
```
Sorry I missed this one 😅